guides/learning-and-development.md

```diff
@@ -0,0 +1,128 @@
+---
+title: Learning and development
+---
+
+We have an annual learning and development allowance of £750 and up to 4 days
+per person.
+
+You can use this for professional and personal development activities like
+workshops, short courses or conferences.
```

Comment:
I wonder if we can add learning platform subscriptions. I know few people that use the budget to subscribe to training platforms 'Skills Matter' or 'Linda..'
And if it's a wellbeing thing can a person use part of the budget for example 'yoga subscription or meditation app' ?
I might be going too wild here..